bokeh/models/tools.py

```diff
@@ -466,8 +466,8 @@ class WheelZoomTool(Scroll):
 
     maintain_focus = Bool(default=True, help="""
     Whether or not zooming tool maintains its focus position. Setting to False
-    results in a more "gliding" behavior, allowing one to zoom out more
-    smoothly, at the cost of losing the focus position.
+    allows zoom-out to happen even at the boundaries of the graph by applying
+    necessary panning in addition to the zoom out, respecting the boundaries.
```

Comment:
I don't like the verbiage here about "applying necessary panning" because that's not at all what is going on (same for text below). What I would like to convey in both cases is this:
> if True, hitting bounds in any on dimensions prevents further zooming in any dimension. If False, zooming will continue in any dimension that has not reached bounds, even if that causes focus or aspect ratio to change.

Reply:
I used your suggestion on both maintain_focus help strings now.